EIPS/eip-1123.md

```diff
@@ -3,13 +3,15 @@ eip: 1123
 title: Revised Ethereum Smart Contract Packaging Standard
 author: g. nicholas d’andrea (@gnidan), Piper Merriam (@pipermerriam), Nick Gheorghita (@njgheorghita), Danny Ryan (@djrtwo)
 discussions-to: https://github.com/ethereum/EIPs/issues/1123
-status: Draft
+status: Abandoned
 type: Standards Track
 category: ERC
 created: 2018-06-01
 replaces: 190
+superseded-by: 2678
```

Comment:
```suggestion
```

`superseded-by` is only valid on `Final` EIPs.

Reply:
Ahh, ok understood, thanks @MicahZoltu !